components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/ui/wizard/CloudConnectorImportWizard.java

```diff
@@ -86,9 +93,14 @@ public boolean performFinish() {
             }
         } else if (removeWizardPage.equals(getContainer().getCurrentPage())) {
             return performFinishRemove();
+        } else if (connectorExporterWizardPage.equals(getContainer().getCurrentPage())) {
+            return performFinishAddToExporter();
         }
         return true;
     }
+    
```

Comment:
shall we remove the additional new lines?

Reply:
fixed https://github.com/wso2/integration-studio/pull/1225/commits/32c29dcb1b7672b6d4b7e47378aa9862f28bf37c